src/Microsoft.Azure.SignalR.Serverless.Protocols/MessagePackServerlessProtocol.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+
+using MessagePack;
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols
+{
+    public class MessagePackServerlessProtocol : IServerlessProtocol
+    {
+        public int Version => 1;
```

Comment:
>Version [](start = 19, length = 7)

Just question, how will we use the Version? In case the parse method have an update in the future? Will this also supported in RT?

Reply:
That's really a problem, we don't have a mechanism like handshake to negotiate a version like traditional signalr.

Reply:
Maybe add a TODO comment in the interface for future discussion?

---
In reply to: [380465249](https://github.com/Azure/azure-functions-signalrservice-extension/pull/92#discussion_r380465249) [](ancestors = 380465249)